torchaudio/backend/sox_io_backend.py

```diff
@@ -14,11 +14,11 @@ def info(filepath: str) -> AudioMetaData:
 
     Args:
         filepath (str or pathlib.Path):
-        Path to audio file. This function also handles ``pathlib.Path`` objects, but is annotated as
-        ``str`` for TorchScript compiler compatibility.
+            Path to audio file. This function also handles ``pathlib.Path`` objects,
+            but is annotated as ``str`` for TorchScript compatibility.
```

Comment:
looks like pathlib is the new standard, so could extend to where it also supported elsewhere